pom.xml

```diff
@@ -69,12 +69,12 @@
     <dependency>
```

Comment:
Shall we also change the client version.  <aws.kinesis.client.version>1.8.10</aws.kinesis.client.version>

Reply:
never mind. I will make the change as part of another commit